clang/docs/ClangFormatStyleOptions.rst

```diff
@@ -6237,18 +6237,30 @@ the configuration (without a prefix: ``Auto``).
        true:                                  false:
        x = ( int32 )y                 vs.     x = (int32)y
 
-  * ``bool InEmptyParentheses`` Put a space in parentheses only if the parentheses are empty i.e. '()'
+  * ``bool InEmptyParentheses`` Put a space in parentheses and braces only if they are empty i.e. '()' or '{}'
```

Comment:
it feels to me like InEmptyParentheses should be parenthese only but InEmptyBraces should inherit by default from InEmptyParentheses  if its not defined in the config?

Reply:
> it feels to me like InEmptyParentheses should be parenthese only

I totally agree with you, but it seems to break compatibility. I didn't know how such a change should land, but I'm eager to implement if it's ok to change that.

> InEmptyBraces should inherit by default from InEmptyParentheses if its not defined in the config?

That sounds reasonable and consistent. But that change would be non-functional and I can't find how I create inheritance of options appropriately. So if it's necessary, could you kindly tell me where inheritance should be placed? 🙏